### ISSUE TITLE
Performance data from host check is missing

### ISSUE THREAD

**Issue by msafdal** [opsdis/icinga2-exporter#24]
Looks like only performance data from services are fetched.

https://github.com/opsdis/icinga2-exporter/blob/eb90b4ce0616efeaecb471db4c8944137d6faac7/icinga2_exporter/monitorconnection.py#L91

Need to export performance data for the host check as well. Available from API endpoint `/v1/objects/hosts?host=host.example.com`. Attrs should be the same as for the services apart from `host_name` which is `name` instead.

```
curl -s -k -H 'X-HTTP-Method-Override: GET' -H 'content-type: application/json' -H 'accept: application/json' -u user:password https://localhost:5665/v1/objects/hosts?host=host.example.com -d '{"attrs": ["__name", "display_name", "check_command", "last_check_result", "vars", "name", "downtime_depth", "acknowledgement", "max_check_attempts", "last_reachable", "state", "state_type"] }' | jq
{
  "results": [
    {
      "attrs": {
        "__name": "host.example.com",
        "acknowledgement": 0,
        "check_command": "hostalive",
        "display_name": "host.example.com",
        "downtime_depth": 0,
        "last_check_result": {
          "active": true,
          "check_source": "host.example.com",
          "command": [
            "/usr/lib/nagios/plugins/check_ping",
            "-H",
            "10.0.30.11",
            "-c",
            "5000,100%",
            "-w",
            "3000,80%"
          ],
          "execution_end": 1630593397.788697,
          "execution_start": 1630593393.700949,
          "exit_status": 0,
          "output": "PING OK - Packet loss = 0%, RTA = 0.47 ms",
          "performance_data": [
            "rta=0.469000ms;3000.000000;5000.000000;0.000000",
            "pl=0%;80;100;0"
          ],
          "schedule_end": 1630593397.788903,
          "schedule_start": 1630593393.7,
          "scheduling_source": "host.example.com",
          "state": 0,
          "ttl": 0,
          "type": "CheckResult",
          "vars_after": {
            "attempt": 1,
            "reachable": true,
            "state": 0,
            "state_type": 1
          },
          "vars_before": {
            "attempt": 1,
            "reachable": true,
            "state": 0,
            "state_type": 1
          }
        },
        "last_reachable": true,
        "max_check_attempts": 3,
        "name": "host.example.com",
        "state": 0,
        "state_type": 1,
        "vars": null
      },
      "joins": {},
      "meta": {},
      "name": "host.example.com",
      "type": "Host"
    }
  ]
}
```

**Comment by thenodon** [comment]
The metrics for host checks will be named with the check command, the same as for services. The service label value for the host metrics will by default be "alive" but can be customized in the configuration file.
